### PR TITLE
new font glyphicon and layer-tool updated

### LINKED ISSUE #52: Some Icons have no coherent font size or white space at top or bottom
The size of the icons is not coherent for all the icons with the same font size. 
Somewere, like in  `eye-close` and `eye-open` icons for instance, there are white spaces on top and bottom that doesn't allow to use them correctly.

See in red the border of the span that contains the char. it causes line breaks. 
image

The eye have 35px font size while the folder is 23px

### PR DESCRIPTION
**Pull request by andreabiagi**
fix #52
New font added.
Wrong icons resize

### CODE REVIEW

File: theme/default/css/font.css [hunk 1]
```diff
@@ -85,13 +86,10 @@
...
   content: "\70";
 }
-.glyphicon-1-max-size:before {
-  content: "\71";
-}
 .glyphicon-1-measure-area:before {
   content: "\72";
 }
-.glyphicon-1-measure-lenght:before {
+.glyphicon-1-measure-leight:before {
```
Comment (Gnafu) [inline theme/default/css/font.css]: is this a typo?

File: theme/default/css/font.css [hunk 2]
```diff
@@ -121,9 +119,6 @@
 .glyphicon-1-screen-back:before {
   content: "\42";
 }
-.glyphicon-1-screen-backward:before {
```
Comment (Gnafu) [inline theme/default/css/font.css:124]: Do not remove, we are using this class

**Comment by offtherailz** [comment]
will this work with this changes ? 
https://github.com/geosolutions-it/MapStore2-theme/pull/53/files#diff-7fdc81d183fb21e4b391fd2356f17f2aR1999

File: theme/default/css/font.css [hunk 3]
```diff
@@ -619,10 +584,39 @@
 .glyphicon-zoom-out:before {
   content: "\e062";
 }
-/** FIXES FOR BOOTSTRAP ICONS **/
-.glyphicon-option-horizontal:before {
-  content: "\77"
+.glyphicon-eye-close:before {
+  content: "\2f";
 }
-.glyphicon-menu-hamburger:before {
```
Comment (Gnafu) [inline theme/default/css/font.css:626]: Do not remove

File: theme/default/css/font.css [hunk 4]
```diff
@@ -619,10 +584,39 @@
 .glyphicon-zoom-out:before {
   content: "\e062";
 }
-/** FIXES FOR BOOTSTRAP ICONS **/
-.glyphicon-option-horizontal:before {
```
Comment (Gnafu) [inline theme/default/css/font.css:623]: Do not remove

**Comment by offtherailz** [comment]
The folder and eye open have 2 different font sizes but seem equal. Keep the size consistent with each other.

**Pull request merged**